inst/include/mode/stepper.hpp

```diff
@@ -151,6 +152,11 @@ class stepper {
     m.rhs(t, y, k1);
   }
 
+  void update_stochastic(double t, rng_state_type& rng_state) {
+    m.update_stochastic(t, y, rng_state);
+    initialise(t); // must recalculate dydt at this point
```

Comment:
I know this wasn't introduced here, but "initialise" still doesn't seem quite right for a name does it? Maybe just "evaluate" or "rhs" or something?

Reply:
this tripped me up when reading it - it definitely needs work. This is a bit everywhere at the moment so have moved into a separate issue so we can see the effect more clearly: https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-3173